sdc/tests/test_dataframe.py

```diff
@@ -1236,13 +1237,27 @@ def test_impl(A):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df2))
 
-    @skip_numba_jit
-    def test_df_reset_index1(self):
-        def test_impl(df):
-            return df.reset_index(drop=True)
+    def test_df_reset_index_drop(self):
+        def test_impl(df, drop):
+            return df.reset_index(drop=drop)
 
         df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0]})
         hpat_func = self.jit(test_impl)
+
+        for drop in [True, False]:
+            with self.subTest(drop=drop):
+                with self.assertRaises(Exception) as raises:
+                    hpat_func(df, drop)
+                msg = 'reset_index Supported only literally drop.'
+                self.assertIn(msg.format(types.bool_), str(raises.exception))
+
+    def test_df_reset_index_drop_false_index_int(self):
+        def test_impl(df):
+            return df.reset_index(drop=False)
+
+        df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0]}, index=[5, 8, 4, 6])
```

Comment:
Testing single columns is obviously not enough to verify implementation is correct.